model/transaction.go

```diff
@@ -145,3 +156,31 @@ func (m TransactionMark) fields() common.MapStr {
 	}
 	return out
 }
+
+type DroppedSpanStats struct {
+	Type                       string
+	Subtype                    string
+	DestinationServiceResource string
+	Outcome                    string
+	Count                      *int
+	DurationSumUs              *int // microseconds
```

Comment:
It would be nice if we could use the recently introduced `model.AggregatedDuration` hwere. I've asked about changing the fields to accommodate that: https://github.com/elastic/apm-server/issues/5850#issuecomment-924627435